File: 3sem/4lab/Banks.Console/Handlers/LastHandler.cs
using Banks.Services;

namespace Banks.Console.Handlers;

public class LastHandler : Handler
{
    public override void Handle(string request, CentralBank centralBank)
    {
        ArgumentNullException.ThrowIfNull(request);
        ArgumentNullException.ThrowIfNull(centralBank);

        System.Console.WriteLine($"'{request}' is invalid request, try 'help' command");
    }
}

File: 4sem/1lab/src/main/java/BanksRemake/Presentation/Handlers/LastHandler.java
package banksRemake.presentation.handlers;

import banksRemake.domain.services.CentralBank;

import java.util.Scanner;

public class LastHandler extends HandlerBase {
    @Override
    public void handle(String request, CentralBank centralBank, Scanner scanner) {
        System.out.println(String.format("'%s' is invalid request, try 'help' command", request));
    }
}
